### PR TITLE
Update VASTTracker documentation

### PR DESCRIPTION
**Pull request by ZacharieTFR**
### Description

- Improved the VASTTracker documentation about methods available to trigger trackers URI. 
- Also removed old warning 

### Type
- [ ] Breaking change
- [ ] Enhancement
- [ ] Fix
- [x] Documentation
- [ ] Tooling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,14 +67,14 @@ vastParser.parseVAST(vastXml)
 
 ### VASTTracker
 
-To track the execution of an ad use the **VASTTracker**:
+To track the execution of an ad, by calling [VAST tracking elements](https://iabtechlab.com/wp-content/uploads/2019/06/VAST_4.2_final_june26.pdf#page=28), use the **VASTTracker**:
```
Comment (Lotphy) [inline README.md:70]: `To track the execution of an ad, create a **VASTTracker** instance and call the [VAST tracking elements](https://iabtechlab.com/wp-content/uploads/2019/06/VAST_4.2_final_june26.pdf#page=28)`
Semantically better in that order imo.

File: docs/api/vast-tracker.md [hunk 2]
```diff
@@ -744,7 +775,7 @@ vastTracker.on('overlayViewDuration', () => {
 vastTracker.overlayViewDuration();
 ```
 
-### verificationNotExecuted(macros)
+### verificationNotExecuted(macros) <a name="verificationNotExecuted"></a>
```
Comment (Lotphy) [inline docs/api/vast-tracker.md:778]: I noticed that this method has actually 2 params 
`verificationNotExecuted(vendor, macros = {})`

Comment (ZacharieTFR) [reply docs/api/vast-tracker.md:778]: Nice catch!

File: README.md [hunk 3]
```diff
@@ -86,10 +86,6 @@ The API documentation is organized by components:
 * [VASTParser](docs/api/vast-parser.md)
 * [VASTTracker](docs/api/vast-tracker.md)
 
-**:warning: IMPORTANT :warning:** : the release of the `3.0` version of the library introduced many breaking changes in the API.
-
-Read the [3.0 migration guide](docs/api/3.0-migration.md) to update your project or follow the [2.0 migration guide](docs/api/2.0-migration.md) or [1.x API documentation](docs/api/1.x) if you're still using the old version.
-
```
Comment (GuillaumeBAECHLER) [inline README.md:92]: I think we should keep an indirect link to these files inside README, with the release entry point for example ?
I'm thinking of something like :

### Changelog 
Changelog & migration guides can be found in the [release notes](https://github.com/dailymotion/vast-client-js/releases).

**Review by GuillaumeBAECHLER — APPROVED** [review]
:shipit:

**Review by Lotphy — APPROVED** [review]

**Pull request merged**